### PR TITLE
10 cards LCI

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/cenote_scout.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Cenote Scout
+ManaCost:G
+Types:Creature Merfolk Scout
+PT:1/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExplore | TriggerDescription$ When CARDNAME enters the battlefield, it explores. (Reveal the top card of your library. Put that card into your hand if it's a land. Otherwise, put a +1/+1 counter on this creature, then put the card back on top or into your graveyard.)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/cenote_scout.txt:5]: reminder text is different between trigger and Oracle

File: forge-gui/res/cardsfolder/upcoming/kinjalli's_dawnrunner.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Kinjalli's Dawnrunner
+ManaCost:2 W
+Types:Creature Bird
+PT:1/1
+K:Double Strike
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/kinjalli's_dawnrunner.txt:5]: Double Strike here but not in Oracle?

File: forge-gui/res/cardsfolder/upcoming/ojer_taq_deepest_foundation_temple_of_civilization.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
+Name:Ojer Taq, Deepest Foundation
+ManaCost:4 W W
+Types:Legendary Creature God
+PT:6/6
+K:Vigilance
+R:Event$ CreateToken | ActiveZones$ Battlefield | ValidToken$ Creature.YouCtrl | ReplaceWith$ TripleToken | Description$ If one or more creature tokens would be created under your control, three times that many of those tokens are created instead.\nWhen Ojer Taq dies, return it to the battlefield tapped and transformed under its owner’s control.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ojer_taq_deepest_foundation_temple_of_civilization.txt:6]: Bad Desc

File: forge-gui/res/cardsfolder/upcoming/ojer_taq_deepest_foundation_temple_of_civilization.txt [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+SVar:TripleToken:DB$ ReplaceToken | Type$ Amount | Amount$ Thrice | ValidCard$ Creature.YouCtrl
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigTransform | TriggerDescription$ When NICKNAME dies, return it to the battlefield tapped and transformed under its owner's control.
+SVar:TrigTransform:DB$ ChangeZone | Defined$ TriggeredCard | Origin$ Graveyard | Destination$ Battlefield | Transformed$ True | Tapped$ True
+DeckHints:Ability$Token
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ojer_taq_deepest_foundation_temple_of_civilization.txt:10]: hmmm maybe Needs on this

File: forge-gui/res/cardsfolder/upcoming/aclazotz_deepest_betrayal_temple_of_the_dead.txt [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ManaCost:3 B B
+Types:Legendary Creature Bat God
+PT:4/4
+K:Flying
+K:Lifelink
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigDiscard | TriggerDescription$ Whenever NICKNAME attacks, each opponent discards a card. For each opponent who can't, you draw a card.
+SVar:TrigDiscard:DB$ Discard | Mode$ TgtChoose | Defined$ Opponent | NumCards$ 1 | RememberDiscarded$ True | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | NumCards$ X | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:Y:Remembered$Amount
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/aclazotz_deepest_betrayal_temple_of_the_dead.txt:11]: `SVar:X:PlayerCountOpponents$Amount/Minus.Remembered$Amount`

File: forge-gui/res/cardsfolder/upcoming/saheeli_the_suns_brilliance.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Name:Saheeli, the Sun's Brilliance
+ManaCost:U R
+Types:Creature Human Artificer
+PT:2/2
+A:AB$ CopyPermanent | Cost$ U R T | ValidTgts$ Creature.YouCtrl+Other,Artifact.YouCtrl+Other | TgtPrompt$ Select another target artifact or creature you control | AddTypes$ Artifact | PumpKeywords$ Haste | AtEOT$ Sacrifice | SpellDescription$ Create a token that's a copy of another target creature or artifact you control, except it's an artifact in addition to its other types. It gains haste. Sacrifice it at the beginning of the next end step.
+DeckHas:Ability$Token|Sacrifice
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/saheeli_the_suns_brilliance.txt:6]: & Type$Artifact

**Review by tool4ever — APPROVED** [review]

**Pull request merged**